### ISSUE TITLE
access targeted view in onGet OR update/updateView not being called for Attribute and Property Bindings

### ISSUE THREAD

**Issue by theladyjaye** [NYTimes/backbone.stickit#140]
TL;DR
1) Do `update` or `updateView` get called for Attribute and Property Bindings? If so, where should they be defined in the binding setup?

2) Is it worth a Pull Request which provides the currently targeted element in `onGet`?

I have a case where I have a list of terms:

``` html
<ul id="foo">
  <li class="term1">Term 1</li>
  <li class="term2">Term 2</li>
  <li class="term3">Term 3</li>
</ul>
```

One of the terms needs to be "selected" based on the content of the model, lets call that attribute on the model `term`.

So my binding is set like this (Yes, there is intentionally duplicate `update` and `updateView` options set. This is an example only, read on for why):

``` javascript
bindings: {
    '#foo li': {
        attributes: [{
            name: 'class',
            observe: 'term',
            onGet: 'formatTermClass',
            updateView: 'shouldUpdateTermClass',
            update: function(){
                console.log('UPDATE INNER');
            }
        }],
        updateView: 'shouldUpdateTermClass',
        update: function(){
            console.log('UPDATE OUTER');
        }
    }
}
```

Now, my `onGet` (formatTermClass) does indeed get called, and I just return 'selected'. Unfortunately, I don't have access in `onGet` to the currently targeted element the binding is going to affect. However, all three of my li's do get a 'selected` class added. This is understandable,  even though it's wrong for my use case, since I'm not doing any logic to determine which one should actually get the 'selected' class. It shows however, that the binding is finding all three of the li's because it is applying the class to all three.

Now, upon further reading, it looks like the preferred way to handle this would be to add a custom `update` or a `updateView`. The `updateView` seems to be the preferable place for some logic like this.

Anyway, as you can see above, I have both `update` and `updateView` defined in the 2 locations that it could possibly picked up (assuming I read the docs correctly). Enter my problem, none of them appear to be called.
